draft-ietf-capport-architecture.xml

```diff
@@ -496,6 +496,189 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
     </section>
    </section>
 
+   <section anchor="ue_identity" title="User Equipment Identity">
+       <t>
+           Multiple components in the architecture interact with both the User
+           Equipment and each other. Since the User Equipment is the focus of
+           these interactions, the components must be able to both identify the
+           user equipment from their interactions with it, and be able to agree
+           on the identify of the user equipment when interacting with each
+           other.
+       </t>
+       <t>
+           The methods by which the components interact restrict the type of
+           information that may be used as an identifying charactertics. This
+           section discusses the identifying charactersitics.
+       </t>
+       <section anchor="id_identifiers" title="Identifiers">
+           <t>
+               An Identifier is an identifying chatacteristics of the User
```

Comment:
I don't think that you should use "identifying" in the definition of "identifier".

Suggest: "An identifier is a characteristic of the UE [...]"

Reply:
Circular definitions for the win. :) I'll use your suggestion.